plugins/runtime/threads/thread-dump-modal.component.ts

```diff
@@ -0,0 +1,31 @@
+namespace Runtime {
+
+  export class ThreadDumpModalController {
+    resolve: { dumpedThreads: string };
+
+    get dumpedThreads() {
+      return this.resolve.dumpedThreads;
```

Comment:
Sanitise the output here using `Core.escapeHtml()`. Otherwise it'll be vulnerable to HTML injection and XSS.

If you are not sure how to use it, search inside the hawtio-integration code and learn from them.